corehq/apps/app_manager/models.py

```diff
@@ -2033,6 +2033,26 @@ def has_persistent_tile(self):
         """
         return self.persist_tile_on_forms and (self.use_case_tiles or self.custom_xml)
 
+    def overwrite_from_module_detail(self, src_module_detail_type, attr_dict):
+        """
+        This method is used to overwrite configurations present
+        in attr_dict(column, filter, and other_configurations)
+        from source module to current object.
+        """
+        src_module_attrs = list(src_module_detail_type._obj.keys())
```

Comment:
Reminder that we talked about replacing `_obj` with a call to `to_json`. (You may remember this, but I forgot so I wanted to write it down.)